bin/tmux_ssh_runs.py

```diff
@@ -33,7 +33,7 @@ def create_new_session_with_server(name, serverloc, tot, host, port):
         serverloc, tot, host, port)
     os.system(
         "TMUX= tmux new-session -d -s {0} 'python3 {1}'".format(
-            name, servercommand, host))
+            name, servercommand))
```

Comment:
@scarlehoff Should 'host' be in this?  (I don't know where TMUX is used without looking)

Reply:
Nop, probably was copied from some older script where we were keeping track of the host in the tmux session or whatever.
I guess this is one of those cases where the `f"{name} {command}"` is clearly superior to the `"{0} {1}".format(name, command)` version...